src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -1,192 +1,140 @@
-            ingestIndexMotor = new WPI_TalonFX(INDEX_INGEST_MOTOR);
-            feederIndexMotor = new WPI_TalonFX(INDEX_FEEDER_MOTOR);
-            // ingestProximity = new DigitalInput(INGEST_PROXIMITY);
-            feederProximity = new DigitalInput(FEEDER_PROXIMITY);
-            ingestTopProximity = new DigitalInput(INGEST_PROXIMITY);
-            // ingestBlueColor = new DigitalInput(INGEST_BLUE);
-            // ingestRedColor = new DigitalInput(INGEST_RED);
-            // feederBlueColor = new DigitalInput(FEEDER_BLUE);
-            // feederRedColor = new DigitalInput(FEEDER_RED);
-            // ingestTopBlueColor = new DigitalInput(INGEST_TOP_BLUE);
-            // ingestTopRedColor = new DigitalInput(INGEST_TOP_RED);
-
-        }
-        if (SHOOTER_ENABLED) {
-            flywheelMotor1 = new WPI_TalonFX(FLYWHEEL_1);
-            flywheelMotor2 = new WPI_TalonFX(FLYWHEEL_2);
-            turretMotor = new WPI_TalonFX(TURRET);
-            hoodMotor = new CANSparkMax(HOOD, CANSparkMaxLowLevel.MotorType.kBrushless);
-        }
-        if (DRIVER_VIS_ENABLED) {
-            CameraServer.addCamera(frontCamera);
-            CameraServer.startAutomaticCapture();
-        }
-    }
+    public static final String LIMELIGHT = "limelight", FRONT_CAM = "front";
+
+    // shooter can ids are range 20-29
+    public static final int FLYWHEEL_1 = 20, FLYWHEEL_2 = 21, TURRET = 22, HOOD = 23;
+
+    // intake can ids are range 30-39
+    public static final int INTAKE_MOTOR_1 = 30, INTAKE_MOTOR_2 = 31, INTAKE_SOLENOID_UP = 1,
+            INTAKE_SOLENOID_DOWN = 0;
+
+    // index can ids are range 40-49
+    public static final int INDEX_INGEST_MOTOR = 40, INDEX_FEEDER_MOTOR = 41, INGEST_RED = 0, INGEST_BLUE = 1,
+            INGEST_PROXIMITY = 2, FEEDER_RED = 3, FEEDER_BLUE = 4, FEEDER_PROXIMITY = 5, INGEST_TOP_RED = 6,
+            INGEST_TOP_BLUE = 7,
+            INGEST_TOP_PROXIMITY = 2;
+
+    // climb can ids are range 50-59
+    public static final int CLIMB_DYNAMIC_MOTOR = 50, CLIMB_FIXED_MOTOR = 51, CLIMB_ANGLE_UP_SOLENOID = 7,
+            CLIMB_ANGLE_DOWN_SOLENOID = 8;
+
+    // PDP logging
+    public static final int PDP_ID = 1; // needs to be verified on the bot (Can be found in REV)
+
+    // Simulation stuff
+    // TODO Find more accurate values
+    public static final double SIM_FULL_VELOCITY = 6000 * TalonFXConstants.RPM_TO_VELOCITY;
+
+    /*
+     * public Hardware() {
+     * boolean comp = Robot.getInstance().isCompetition();
+     *
+     * PDP = new PowerDistribution(PDP_ID, ModuleType.kRev);
+     *
+     * if (DRIVE_ENABLED) {
+     * frontLeftModule = FRONT_LEFT_CONFIG.create(comp);
+     * frontRightModule = FRONT_RIGHT_CONFIG.create(comp);
+     * backLeftModule = BACK_LEFT_CONFIG.create(comp);
+     * backRightModule = BACK_RIGHT_CONFIG.create(comp);
+     * gyro = comp ? new Pigeon(GYRO_PORT) : new NavX(SerialPort.Port.kMXP);
+     * }
+     * if (!comp)
+     * return;
+     * if (CLIMB_ENABLED) {
```

Comment:
Please delete the old hardware class code, as it's not needed anymore.

Reply:
Oh yeah, I forgot to do that